src/scripts/q-expand.test.js

```diff
@@ -5,16 +5,43 @@ const script = require("./q-expand");
 
 describe("q-expand", () => {
   const app = getApp();
+  let getCsvData;
+
+  beforeAll(() => {
+    getCsvData = script.getCsvData;
+    script.getCsvData = jest.fn();
+  });
+
+  afterAll(() => {
+    script.getCsvData = getCsvData;
+  });
 
   beforeEach(() => {
     app.brain.clear();
     jest.resetAllMocks();
+
+    script.getCsvData.mockResolvedValue({
+      Q: "Top level",
+      QQ: "One depth",
+      QQC: "Not a contractor",
+      QU: "One nest",
+      QUE: "A second",
+      QUEA: "Another level!",
+      QUEAA: "Keep going down",
+      QUEAAB: "So far down!",
+      QUEAAA: "Deep down now!",
+      QUEAAC: "Center of the Earth!",
+      QUEAAD: "Whee, way down!",
+      QUEAB: "Secret level",
+      QUEABA: "Secret sublevel 1",
+      QUEABB: "Secret sublevel 2",
+    });
```

Comment:
Setup the mocking. The tests below are updated accordingly to reflect this mocked data rather than the real data from the CSV.